docs/_edot-sdks/nodejs/migration.md

```diff
@@ -7,8 +7,202 @@ parent: EDOT Node.js
+
+For example: `OTEL_BSP_MAX_QUEUE_SIZE=4096`.
+
+### `serverTimeout`
+
+The Elastic [`serverTimeout`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#server-timeout) option corresponds to the OpenTelemetry [`OTEL_BLRP_EXPORT_TIMEOUT`](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#batch-span-processor) option.
+
+For example: `OTEL_BLRP_EXPORT_TIMEOUT=50000`.
+
+### `apmClientHeaders`
+
+The Elastic [`apmClientHeaders`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#apm-client-headers) option corresponds to the OpenTelemetry [`OTEL_EXPORTER_OTLP_HEADERS`](https://opentelemetry.io/docs/specs/otel/protocol/exporter/#specifying-headers-via-environment-variables) option.
+
+For example: `OTEL_EXPORTER_OTLP_HEADERS=foo=bar,baz=quux`.
+
+### `disableInstrumentations`
+
+The Elastic [`disableInstrumentations`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#apm-client-headers) option corresponds to the OpenTelemetry [`OTEL_NODE_DISABLED_INSTRUMENTATIONS`](https://opentelemetry.io/docs/zero-code/js/configuration/#disable-specific-instrumentations) option. See [Configuration](./configuration) if you want to learn more advanced usage.
+
+For example: `OTEL_NODE_DISABLED_INSTRUMENTATIONS=express,mysql`.
+
+### TODO (double check) `containerId`
+
+The Elastic [`containerId`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#container-id) option corresponds to setting the `container.id` key in [OTEL_RESOURCE_ATTRIBUTES](https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_resource_attributes).
+
+For example: `OTEL_RESOURCE_ATTRIBUTES=container.id=my-id`.
+
+### `metricsInterval`
+
+The Elastic [`metricsInterval`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#metrics-interval) option corresponds to the OpenTelemetry [`OTEL_METRIC_EXPORT_INTERVAL`](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#periodic-exporting-metricreader) option.
+
+For example: `OTEL_METRIC_EXPORT_INTERVAL=30000`.
+
+### `cloudProvider`
+
+The Elastic [`cloudProvider`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#cloud-provider) option does not corresponds directly to an OpenTelemetry option but you can get similar behaviour by properly setting [`OTEL_NODE_RESOURCE_DETECTORS`](https://opentelemetry.io/docs/zero-code/js/configuration/#sdk-resource-detector-configuration) option. You have to make sure the
+cloud provider is on the list of detectors. Not setting this option is the equivalent of `auto`.
+
+For example: `OTEL_NODE_RESOURCE_DETECTORS=os,env,host,process,gcp`. Will make the agent query for GCP metadata only.
```

Comment:
This is missing some non-cloud detectors, I think. `serviceinstance` and `container`. Could refer to https://elastic.github.io/opentelemetry/edot-sdks/nodejs/configuration.html#otel_node_resource_detectors-details instead of the opentelemetry.io link.